stock_reserve/data/stock_data.xml

```diff
@@ -3,6 +3,7 @@
     <data noupdate="1">
         <record id="stock_location_reservation" model="stock.location">
             <field name="name">Reservation Stock</field>
+            <field name="reserved">True</field>
             <field name="location_id" ref="stock.stock_location_locations"/>
```

Comment:
If it is standard, forget about it
It is strange though because I thought ids wouldnot have plurals